src/components/course/CourseSidebarPrice.jsx

```diff
@@ -74,27 +67,10 @@ const CourseSidebarPrice = () => {
 
   // Case 3: No subsidies found
   if (!hasDiscountedPrice) {
-    const { catalogList } = courseData.catalog;
-    const hasOfferWithInsufficientBalance = canEnrollWithEnterpriseOffers && enterpriseOffers.find(
-      (enterpriseOffer) => {
-        const isCourseInCatalog = catalogList.includes(enterpriseOffer.enterpriseCatalogUuid);
-        const hasInsufficientBalance = offerHasBookingsLimit(enterpriseOffer)
-          && enterpriseOffer.remainingBalance < coursePrice.list;
-
-        return isCourseInCatalog && hasInsufficientBalance;
-      },
-    );
-
     return (
-      <>
-        <span className="d-block">${originalPriceDisplay} {currency}</span>
-        {hasOfferWithInsufficientBalance && (
-          <small data-testid="insufficient-offer-balance-text">
-            {INSUFFICIENT_ENTERPRISE_OFFER_BALANCE}{' '}
-            <ContactAdminMailto /> to learn more.
-          </small>
-        )}
```

Comment:
This "insufficient offer balance" messaging is no longer desired, in favor of simply showing the original list price.